woocommerce/woocommerce-gateway-amazon-pay
# Update dependencies

### All Submissions:


### Changes proposed in this Pull Request:

Update our npm dependencies. This PR replaces the following PRs that were opened by dependabot:

- #2 
- #3 
- #88 
- #92 
- #95 
- #96 
- #141 
- #154 
- #156 

### How to test the changes in this Pull Request:

1. Ensure you are running node v10 and npm v6 or below, as the way we are building assets is no longer supported as of [npm 7](https://github.com/npm/cli/blob/latest/changelogs/CHANGELOG-7.md#all-lifecycle-scripts). No changes to this were made in this PR, this is existing functionality, but would be nice to modernize this at some point
2. On a clean install, run `npm install`
3. Then run `npm run build:assets`
4. Ensure all CSS, minified JS and the POT files are created properly
5. Ensure all functionality that relies on the CSS or JS files still works

### Other information:


### Changelog entry

> Update all npm dependencies
